tests/foreman/cli/test_activationkey.py

```diff
@@ -292,11 +292,12 @@ def test_negative_create_with_invalid_name(self):
             if len(name) == 0:
                 content_to_validate.append(
                     u'Name must contain at least 1 character')
-            elif len(name) > 255:
+            if len(name) > 255:
                 content_to_validate.append(
                     u'Name is too long (maximum is 255 characters)')
-            else:
-                content_to_validate.append(name)
+            if name == "\t":
```

Comment:
doesn't seem that `==\t` really verify that name has no characters as it will not get through `==\n` or any other special characters. Maybe we can find other option

Reply:
thanks @oshtaier for pointing that out.